tests/integration/targets/playbooks/provision_playbook/enterprise-example.com/platform/kong/prod-kong/secret-ee-admin-gui-auth.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   name: tavros-prod-kong-ee-admin-gui-auth
   namespace: kong
 data:
-  admin_gui_auth_conf: eyJpc3N1ZXIiOiAiaHR0cHM6Ly9hdXRoLnRhdnJvcy5lbnRlcnByaXNlLWV4YW1wbGUuY29tL2F1dGgvcmVhbG1zL3Byb2QiLCAiY2xpZW50X2lkIjogWyJwcm9kLWtvbmciXSwgImNsaWVudF9zZWNyZXQiOiBbInBsYWNlaG9sZGVyMTIiXSwgImNvbnN1bWVyX2J5IjogWyJ1c2VybmFtZSIsICJjdXN0b21faWQiLCAiaWQiXSwgInNzbF92ZXJpZnkiOiBmYWxzZSwgImNvbnN1bWVyX2NsYWltIjogWyJlbWFpbCJdLCAibGVld2F5IjogMTAwMCwgInJlZGlyZWN0X3VyaSI6IFsiaHR0cHM6Ly9hZG1pbi5wcm9kLWtvbmcudGF2cm9zLmVudGVycHJpc2UtZXhhbXBsZS5jb20iXSwgImxvZ2luX3JlZGlyZWN0X3VyaSI6IFsiaHR0cHM6Ly9hZG1pbi5wcm9kLWtvbmcudGF2cm9zLmVudGVycHJpc2UtZXhhbXBsZS5jb20iXSwgImxvZ291dF9tZXRob2RzIjogWyJHRVQiLCAiREVMRVRFIl0sICJsb2dvdXRfcXVlcnlfYXJnIjogImxvZ291dCIsICJsb2dvdXRfcmVkaXJlY3RfdXJpIjogWyJodHRwczovL2FkbWluLnByb2Qta29uZy50YXZyb3MuZW50ZXJwcmlzZS1leGFtcGxlLmNvbSJdLCAic2NvcGVzIjogWyJvcGVuaWQiLCAicHJvZmlsZSIsICJlbWFpbCIsICJvZmZsaW5lX2FjY2VzcyJdLCAiYXV0aF9tZXRob2RzIjogWyJhdXRob3JpemF0aW9uX2NvZGUiXX0=
+  admin_gui_auth_conf: ewogICAgImlzc3VlciI6Imh0dHBzOi8vYXV0aC50YXZyb3MuZW50ZXJwcmlzZS1leGFtcGxlLmNvbS9hdXRoL3JlYWxtcy9wcm9kIiwKICAgICJjbGllbnRfaWQiOlsicHJvZC1rb25nIl0sCiAgICAiY2xpZW50X3NlY3JldCI6WyJwbGFjZWhvbGRlcjEyIl0sCiAgICAiY29uc3VtZXJfYnkiOlsidXNlcm5hbWUiLCJjdXN0b21faWQiLCJpZCJdLAogICAgInNzbF92ZXJpZnkiOmZhbHNlLAogICAgImNvbnN1bWVyX2NsYWltIjpbImVtYWlsIl0sCiAgICAibGVld2F5IjoxMDAwLAogICAgInJlZGlyZWN0X3VyaSI6WyJodHRwczovL2FkbWluLnByb2Qta29uZy50YXZyb3MuZW50ZXJwcmlzZS1leGFtcGxlLmNvbSJdLAogICAgImxvZ2luX3JlZGlyZWN0X3VyaSI6WyJodHRwczovL2FkbWluLnByb2Qta29uZy50YXZyb3MuZW50ZXJwcmlzZS1leGFtcGxlLmNvbSJdLAogICAgImxvZ291dF9tZXRob2RzIjpbIkdFVCIsICJERUxFVEUiXSwKICAgICJsb2dvdXRfcXVlcnlfYXJnIjoibG9nb3V0IiwKICAgICJsb2dvdXRfcmVkaXJlY3RfdXJpIjpbImh0dHBzOi8vYWRtaW4ucHJvZC1rb25nLnRhdnJvcy5lbnRlcnByaXNlLWV4YW1wbGUuY29tIl0sCiAgICAic2NvcGVzIjpbIm9wZW5pZCIsInByb2ZpbGUiLCJlbWFpbCIsIm9mZmxpbmVfYWNjZXNzIl0sCiAgICAiYXV0aF9tZXRob2RzIjpbImF1dGhvcml6YXRpb25fY29kZSJdCn0KICAgIAo=
```

Comment:
I don't think these should we changing...

Reply:
They shouldn't but this is related to the weird errors that came from nowhere. Im still trying to resolve this actually.